components/gcp/container/component_sdk/python/kfp_component/google/bigquery/_query.py

```diff
@@ -24,6 +24,40 @@
 # TODO(hongyes): make this path configurable as a environment variable
 KFP_OUTPUT_PATH = '/tmp/kfp/output/'
 
+
+def query_only(query, project_id, output_path, dataset_location='US', job_config=None):
```

Comment:
Do you mind explaining why this function is named `query_only` while it's actually dumping a local file?

Reply:
I removed this function completely.